alpha/apps/kaltura/lib/model/objectfilters/UserEntryFilter.php

```diff
@@ -28,6 +28,7 @@ protected function init()
 			"_eq_extended_status",
```

Comment:
Now that extended_status is not part of the basic userEntry and you moved it to  ViewHistoryUserEntryFilter.php it should not be here